src/pages/contacts-page/contacts-edit/contacts-edit.js

```diff
@@ -170,192 +159,13 @@ const ContactsEdit = ({ match }) => {
-                  label='Розклад (англ.)'
-                  multiline
-                  InputLabelProps={{
-                    classes: {
-                      root: classes.inputLabel,
-                      shrink: 'shrink'
-                    }
-                  }}
-                  value={values.enSchedule}
-                  onChange={handleChange}
-                  error={touched.enSchedule && !!errors.enSchedule}
-                  helperText={touched.enSchedule && errors.enSchedule}
-                  required
-                />
-              </Paper>
-            </Grid>
-            <Grid item xs={6}>
-              <Paper className={classes.newsItemUpdate}>
-                <TextField
-                  id='ukAddress'
-                  className={classes.textField}
-                  variant='outlined'
-                  label='Адреса (укр.)'
-                  multiline
-                  InputLabelProps={{
-                    classes: {
-                      root: classes.inputLabel,
-                      shrink: 'shrink'
-                    }
-                  }}
-                  value={values.ukAddress}
-                  onChange={handleChange}
-                  error={touched.ukAddress && !!errors.ukAddress}
-                  helperText={touched.ukAddress && errors.ukAddress}
-                  required
-                />
-                <TextField
-                  id='enAddress'
-                  className={classes.textField}
-                  variant='outlined'
-                  label='Адреса (англ.)'
-                  multiline
-                  InputLabelProps={{
-                    classes: {
-                      root: classes.inputLabel,
-                      shrink: 'shrink'
-                    }
-                  }}
-                  value={values.enAddress}
-                  onChange={handleChange}
-                  error={touched.enAddress && !!errors.enAddress}
-                  helperText={touched.enAddress && errors.enAddress}
-                  required
-                />
-                <TextField
-                  id='email'
-                  className={classes.textField}
-                  variant='outlined'
-                  label='Email'
-                  InputLabelProps={{
-                    classes: {
-                      root: classes.inputLabel,
-                      shrink: 'shrink'
-                    }
-                  }}
-                  value={values.email}
-                  onChange={handleChange}
-                  error={touched.email && !!errors.email}
-                  helperText={touched.email && errors.email}
-                  required
-                />
-              </Paper>
-            </Grid>
-          </Grid>
-        </FormControl>
-        <SaveButton
-          id='save'
-          type='submit'
-          title='Зберегти'
-          className={classes.saveButton}
-        />
-      </form>
-    </div>
+    <ContactsForm
```

Comment:
It is better to move `useFormik` call to the `ContactForm` component and just pass as a props `initialValues`, `submitHandler`, etc. It is better to have the form-hook inside the actual form component